src/cljs/main/broadfcui/config.cljs

```diff
@@ -65,6 +65,4 @@
 (defn terra-base-url [] (get @config "terraBaseUrl"))
 (defn terra-redirects-enabled [] (get @config "terraRedirectsEnabled" false))
 (defn tos-url [] (get @config "tosUrl"))
```

Comment:
I think I also want to delete `tos-url`, but have a question on the usage below (@MatthewBemis).